pkgs/applications/misc/kitty/default.nix

```diff
@@ -45,6 +45,11 @@ buildPythonApplication rec {
     cp -r linux-package/{bin,share,lib} $out
     wrapProgram "$out/bin/kitty" --prefix PATH : "$out/bin:${stdenv.lib.makeBinPath [ imagemagick xsel ]}"
     runHook postInstall
+
+    mkdir -p "$out/share/"{bash-completion/completions,fish/vendor_completions.d,zsh/site-functions}
+    "$out/bin/kitty" + complete setup fish > "$out/share/fish/vendor_completions.d/kitty.fish"
+    "$out/bin/kitty" + complete setup bash > "$out/share/bash-completion/completions/kitty.bash"
+    "$out/bin/kitty" + complete setup zsh > "$out/share/zsh/site-functions/_kitty"
```

Comment:
It only works when explicitly sourcing the file, but not by putting into fpath:

https://github.com/kovidgoyal/kitty/blob/8ceb941051b89b7c50850778634f0b6137aa5e6e/docs/index.rst#zsh

Just skip zsh for now I would say.

Reply:
yeah let's drop that for now. If somebody needs ZSH completions, it's still possible to source them in `programs.zsh.shellInit`.

Reply:
I updated the install script and left a comment